editor/editor_file_system.cpp

```diff
@@ -822,7 +822,7 @@ void EditorFileSystem::_scan_fs_changes(EditorFileSystemDirectory *p_dir, const
 			}
 		}
 
-		EditorResourcePreview::get_singleton()->check_for_invalidation(p_dir->get_file_path(i));
+		//EditorResourcePreview::get_singleton()->check_for_invalidation(p_dir->get_file_path(i));
```

Comment:
Please remove commented out code, unless there is a real reason to keep it around (in which case an explanatory comment is required).

Reply:
but for every commented line here i removed another one in other place...
ok, i will remove those.